portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,28 @@
   width: 650px;
 }
 
+.w3-cinzel {
```

Comment:
name classes according to what they represent on the page, not according to the details of the styling (e.g. as an example, use .subtitle instead of .small-font)

Reply:
Done